oneview_redfish_toolkit/api/scmb.py

```diff
@@ -51,10 +52,13 @@ def get_oneview_client():
     # Workaround for #328
     # Create OneView client using API 500 just to retrieve OneView certificates
     try:
-        ov_config = deepcopy(util.ov_config)
-        ov_config['api_version'] = 500
+        ov_config = connection.create_oneview_config(
+            ip=config.get_oneview_ip(),
+            credentials=config.get_credentials(),
+            api_version=500
+        )
         ov_client = OneViewClient(ov_config)
-        ov_client.connection.login(ov_config['credentials'])
+        ov_client.connection.login(config.get_credentials())
     # if failed abort
     except Exception:
```

Comment:
This catch is unnecessary because we are doing nothing here, just raising the exception again.

Reply:
Done.